parcels/particleset.py

```diff
@@ -26,6 +29,51 @@
 __all__ = ['ParticleSet']
 
 
+class ParticleAccessor(object):
```

Comment:
That a really need implementation to make the structural change transparent for users.